src/main/drivers/max7456.c

```diff
@@ -171,11 +170,11 @@ void max7456_dma_irq_handler(dmaChannelDescriptor_t* descriptor) {
         SPI_I2S_DMACmd(MAX7456_SPI_INSTANCE,
 #ifdef MAX7456_DMA_CHANNEL_RX
                 SPI_I2S_DMAReq_Rx |
-#endif
+#endif // MAX7456_DMA_CHANNEL_RX
                 SPI_I2S_DMAReq_Tx, DISABLE);
 
         DISABLE_MAX7456;
-        dma_transaction_in_progress = 0;
+        dmaTxInProgress = 0;
```

Comment:
0 and 1 is kind of ugly. I personally prefer either some clear enums or simply bool.